### ISSUE TITLE
ESP8266

### ISSUE THREAD

**Issue by stanzlavos** [marekburiak/ILI9341_due#9]
Is it possible to get this working on a NodeMCU ESP8266 ? :)

**Comment by marekburiak** [comment]
I am sure it is possible. The question is, who's got time to do it? :)

**Comment by stanzlavos** [comment]
Let me rephrase :D

Does the ESP8266 support DMA? In their documentation I did see DMA support
but only in the context of I2S interface.

**Comment by marekburiak** [comment]
Unfortunately I don't know.

**Comment by hasenradball** [comment]
Hi together,
Is there anything new to this topic?
I would like to run the library on an ESP8266 and ESP32.
Does it actually work now?

**Comment by marekburiak** [comment]
nah, don't have time for it at all. Maybe the generic  ILI9341_SPI_MODE_NORMAL works but I haven't tried.

**Comment by hasenradball** [comment]
Hi, no problem. I can try it.
I could also support. Do you have an Idea which special points to look at?

**Comment by marekburiak** [comment]
What do you mean by "support"? And special points for what?

**Comment by hasenradball** [comment]
Sorry Support is the wrong word.
I mean I can have a look at if the ESP works with the library.
But do you know from your experience which points have considered at first if it does not work.

**Comment by marekburiak** [comment]
I guess it depends on what compilation errors you get :)
And if by any chance it compiles fine then we'll see how the screen behaves.

**Comment by fotosettore** [comment]
hi,
any news about development and test of this library in esp32 and/or esp8266 ?

**Comment by marekburiak** [comment]
Hi, nope, nothing new. I have absolutely no time (and honestly also very small desire) to look at it.
